core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -36,28 +37,44 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
               }
             }
         } ~
-          pathPrefix("members") {
-            pathEnd {
-              get {
+        pathPrefix("spendReportConfiguration") {
```

Comment:
If you've made it here, I'd recommend reviewing with whitespace changes turned off